DESCRIPTION

```diff
@@ -40,7 +40,9 @@ Suggests:
   knitr,
   mockery,
   rcmdcheck,
-  rmarkdown
+  rmarkdown,
+  spelling
```

Comment:
Please bump dev version.

Reply:
Bumping it to 9003, but not adding anything in NEWS, since this is an internal check.

Reply:
Please always bump by 1. If you merge a different PR before this one you'll run into a merge conflict anyway - that's when you merge / rebase onto `main` and fix the version :slightly_smiling_face:

Reply:
Actually I'm wrong about merge conflicts: if two PRs bump from e.g. `9001` to `9002`, when the first one is merged, the second one won't show any conflicts, so we'll need to remember to change it accordingly...

Still I'd prefer to keep the "bump by 1" rule, as we won't always know the order of merging of PRs - it is especially true, when multiple developers are working at the same time.

Consider having a separate commit for version bump. Then it'll be easy to perform an interactive rebase onto master to update the version.